deploy/operator.yaml

```diff
@@ -35,6 +35,34 @@ spec:
               value: "oadp-operator"
             - name: ANSIBLE_GATHERING
               value: explicit
+            - name: REGISTRY
+              value: quay.io
+            - name: PROJECT
+              value: konveyor
+            - name: VELERO_REPO
+              value: velero
+            - name: VELERO_PLUGIN_REPO
+              value: migration-plugin   #TODO: Ask if this needs to be changed or not
```

Comment:
Let's leave this here for now, and remove it as an initcontainer since this will change.

Reply:
Updated the value !